src/main/res/values/strings.xml

```diff
@@ -78,24 +78,24 @@
     <string name="link_copied_to_clipboard">Link copied to clipboard</string>
     <string name="links">Links</string>
     <string name="list_animation">List Animations</string>
-    <string name="list_animation_description">Show list animation on the main page</string>
+    <string name="list_animation_description">Animation the list on the main page</string>
     <string name="merging_FORMAT">Merging %s</string>
     <string name="name">Name</string>
     <string name="network_error_DESC">Network error.</string>
     <string name="never">Never</string>
-    <string name="new_updates_available">New versions of applications available</string>
+    <string name="new_updates_available">New versions available</string>
```

Comment:
In plural this can't be Droid-ify itself.
Explained by "app/s "in item quantify